src/client/polyfills/css-shim/test/css-shim.spec.ts

```diff
@@ -366,8 +366,8 @@ describe('css-shim', () => {
     );
   });
 
-  var window: Window;
```

Comment:
I think for our shims, it's safer to leave `var` in place. They (the shims) are designed to run in older browsers (like Safari 10, IE 11). But I think these shims will go away in a future major release where we don't support those browsers. For now, can we ignore 'no-var' here?

Reply:
sure, that makes sense, I'll make that edit now